frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -170,7 +171,10 @@ export const getCurrentVersion = (cv: ClusterVersionKind): string => {
   return _.get(cv, 'status.history[0].version') || _.get(cv, 'spec.desiredUpdate.version');
 };
 
-export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href: string } => {
+export const getReportBugLink = (
+  cv: ClusterVersionKind,
+  t: TFunction,
```

Comment:
I think @spadgett was against passing the TFunction around as a prop. What other folks have done is use `i18next.t` instead. You don't need a component for that particular usage.

Reply:
Hi @rebeccaalpert, thanks I believe I have addressed this issue.

Reply:
@dtaylor113 I still see you passing `t` in the PR.

Reply:
We're still passing it around here.